generated/nidaqmx/system/_collections/device_collection.py

```diff
@@ -7,6 +7,7 @@
 from nidaqmx.error_codes import DAQmxErrors
 from nidaqmx.system.device import Device
 from nidaqmx.utils import unflatten_channel_string
+from nidaqmx.system.device import _DeviceAlternateConstructor
```

Comment:
Style: combine `from X import A` and `from X import B` into `from X import A, B`

Normally ni-python-styleguide would complain that these imports are out of order, but we are still excluding src/handwritten and removing that exclusion will require a lot of changes.